test/models/user_test.rb

```diff
@@ -0,0 +1,56 @@
+require "test_helper"
+
+describe User do
+  describe "validations" do
+    it "can be instantiated" do
+   
+      expect users(:user_1).valid?.must_equal true
+     end
+
+    it "will have the required fields" do
+      users(:user_1).username = nil
+      no_username = users(:user_1).valid?
+       expect _(no_username).must_equal false      
```

Comment:
Instead of using `expect _(...)` you should just use `expect(...)`:

```suggestion
       expect(no_username).must_equal false      
```